test.disabled/ejabberd_tests/tests/muc_SUITE.erl

```diff
@@ -2792,32 +2792,39 @@ exit_room_with_status(Config1) ->
 kicked_after_sending_malformed_presence(Config1) ->
     AliceSpec = given_fresh_spec(Config1, alice),
     Config = given_fresh_room(Config1, AliceSpec, []),
-    escalus:fresh_story(Config, [{bob, 1}, {kate, 1}], fun(Bob, Kate) ->
+    escalus:fresh_story(Config, [{bob, 1}, {kate, 1},{alice, 1}], fun(Bob, Kate, Alice) ->
         %% GIVEN An user is in the room
         Room = ?config(room, Config),
+        KateUsername = escalus_utils:get_username(Kate),
```

Comment:
Why do we need Kate in the test?

Reply:
To prevent room process from terminating when Bob is kicked

Reply:
hmm, maybe the whole test should be something like:

```erlang
kicked_after_sending_malformed_presence(Config) ->
      muc_helper:with_fresh_room(Config, [{alice, 1}], fun(RoomName, Alice) ->
             muc_helper:join(RoomName, Alice),
             muc_helper:send_malformed_presence(RoomName, Alice),
             muc_helper:gets_kicked(RoomName, Alice),
             muc_helper:assert_not_in_the_room(RoomName, Alice)
             end).
```

:)
(this comment should be ignored for this PR).